statscache/schedule.py

```diff
@@ -37,6 +37,12 @@ class Hour(Denomination):
 
 class Schedule(object):
     def __init__(self, second=None, minute=None, hour=None):
+
+        # Let higher order denominations precede lesser ones
+        # https://github.com/fedora-infra/statscache/issues/10
+        second = second or (minute and [0])
+        minute = minute or (hour and [0])
```

Comment:
I wonder if I understand this correctly:
````
>>> None or (2 and [0])
[0]
>>> None or (1 and [0])
[0]
>>> None or ([0])
[0]
````

Isn't the last version either then?